DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/stats/stats.controller.js

```diff
@@ -0,0 +1,50 @@
+﻿(function() {
+    'use strict';
+
+    angular.module('app.stats')
+        .controller('statsController', statsController);
+    statsController.$inject = ['statsFactory', '$scope', 'logger'];
+
+    function statsController(statsFactory, $scope, logger) {
+        $scope.rep = '';
+        $scope.series = ['Series A'];
+
+        getCommits().then(function () {
+            logger.info('Got commits from GitHub!');
```

Comment:
I am not sure that this `logger.info('Got commits from GitHub!');` is executed after service call.